baseband/mark4/header.py

```diff
@@ -301,6 +311,21 @@ def verify(self):
                                                  self['lsb_output']))) ==
                 self.nchan)
 
+    @staticmethod
+    def infer_decade(ref_time, header_unit_year):
```

Comment:
We use `infer_decade` in only two places, in both of which the header's own `header_unit_year` is used. I think this suggests it should just be a regular method, i.e.,
```
def infer_decade(self, ref_time):
    self.decade = ...
```
(and it should be in `Mark4TrackHeader`)